src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -444,6 +415,65 @@ public int Main(string[] args)
                 });
             });
 
+            app.Command("install-g", c =>
```

Comment:
When we talked a few ago, you mentioned that you would prefer to have distinct commands `install` and `uninstall` for these commands (https://github.com/aspnet/XRE/issues/1135). I can go either way, just let me know if which one you prefer. Btw, I think `commands install` is better than `commands add`